src/protog.cpp

```diff
@@ -36,6 +39,21 @@ void print_help(FILE* f) {
     fprintf(f, "  protog -p openrtb.proto -m com.google.openrtb.BidRequest -i openrtb.pb.h\n");
 }
 
+void make_output_dir(const char* output_dir, char* buffer) {
+    struct stat st = {0};
+    if (stat(output_dir, &st) == -1) {
+        mkdir(output_dir, 0700);
```

Comment:
Hmm.. and if it exists we'd need to delete the files that we'll write to? All those little details ^^